application/templates/application/global-html.html

```diff
@@ -133,7 +188,7 @@ <h3>Psychiatric Guide</h3> <!-- Generic Title -->
         </div>
     </center>
 </footer>
-
+-->
```

Comment:
Same here. All of this is getting handled in the sidebar so we should just get rid of it.